.wsignore

```diff
@@ -11,6 +11,12 @@ experimental/python/docs/images/databricks-logo.svg
 **/*.zip
 **/*.whl
 
+# new lines are recorded differently on windows and unix.
+# In unix: "raw_body": "hello, world\n"
+# In windows: "raw_body": "hello, world\r\n"
+# In order to prevent that difference, hello.txt does not have a trailing newline.
+acceptance/selftest/record_cloud/volume-io/hello.txt
```

Comment:
I tried out [[Repls]], but they did not seem to work. I could not reproduce this locally on Windows, and it only seems to happen in the Windows CI runner. 

I've already spent 1-2 hours trying to fix this, so the easiest thing to do at this point is to ignore the lint rule.
I'm not sure we want to normalize in the raw_body since it is a part of the HTTP payload.

Reply:
The file definitely does not have a trailing newline (at least in my local checkout). I'm not sure why it's not being highlighted. Let's see if the latest integration tests pass, in which case maybe github is not rendering it properly.